src/client.ts

```diff
@@ -51,6 +51,16 @@ import {
 import { DeployStatus } from './types/consts';
 import { ClientApiError, convertApiErrorsAndThrow } from './utils/error';
 import { getListOfFunctionsAndAssets, SearchConfig } from './utils/fs';
+import {
+  HTTPAlias,
+  OptionsOfJSONResponseBody,
+  OptionsOfTextResponseBody,
+} from 'got/dist/source';
+import pLimit, { Limit } from 'p-limit';
+
+const RETRY_DEFAULTS = {
+  limit: 10,
```

Comment:
Can we make this configurable through an environment variable or something? This can be an obscure one.

Reply:
Yes, good idea.